modules/go-health-monitoring.adoc

```diff
@@ -20,4 +20,6 @@ The {gitops-title} Operator will install the GitOps backend service in the `open
 . If the application is `out of sync` or `degraded` applications the respecting icons are displayed under the *Resources*. Hover over the icons to see the health status and the sync status. The icons are:
```

Comment:
If the application is out of sync or degraded, the respective icons are displayed under the *Resources* <tab/section???>. To view the sync and health status of the applications, over the cursor over the icons.

Reply:
Suggestion looks good - definitely needs a comma, and dropping "applications" after `degraded` reads better. 

And *Resources* does need something after it (or else drop "the" before it)